src/app/routes.py

```diff
@@ -46,6 +46,18 @@ def get_task():
     return jsonify(status="ok", grid=task)
 
 
+@app.route("/v1/task", methods=["GET"])
+def get_task_difficulty():
+    logger.info("Generating task...")
+    min_difficulty = int(request.args["min_difficulty"])
+    max_difficulty = int(request.args["max_difficulty"])
```

Comment:
```suggestion
    min_difficulty = int(request.args["min_difficulty"])
    max_difficulty = int(request.args["max_difficulty"])
    logger.info("Generating task with min_difficulty = %s, max_difficulty = %s", min_difficulty, max_difficulty )
```

А что, если какой-то из параметров не будет передан? Обязательны ли эти границы?

Reply:
Так этот запрос как раз предназначен для выдачи задания определенной сложности. Если без параметров, то нужно использовать get_task

Reply:
Да, но в то же время `Sudoku.get_task()` позволяет опускать одну из или обе границы. А этот метод не позволяет.